index.js

```diff
@@ -64,6 +66,14 @@ module.exports = (moduleObj, options) => {
 			console.log('File changed:', filePath);
 		}
 
+		const interval = options.interval || 0;
```

Comment:
The option already has a default value set.

Reply:
Oh, it's my bad habit. I have modified it.